src/index.html

```diff
@@ -4,9 +4,355 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__content">
+        <a href="#" class="logo"><img src="./images/logo.png" alt="moyo logo"></a>
```

Comment:
```suggestion
        <a href="#" class="logo">
          <img src="./images/logo.png" alt="moyo logo">
        </a>
```
Pay attention to formatting